seqr/views/apis/data_manager_api_tests.py

```diff
@@ -1708,9 +1712,9 @@ def _has_expected_ped_files(self, mock_open, mock_mkdir, dataset_type, *args, sa
         super()._has_expected_ped_files(mock_open, mock_mkdir, dataset_type,  *args, sample_type, **kwargs)
         mock_mkdir.assert_called_once_with(self._local_pedigree_path(dataset_type, sample_type), exist_ok=True)
 
-    def _assert_success_notification(self, dag_json):
+    def _assert_success_notification(self, dag_json, config_params):
```

Comment:
it appears in both calls to `_assert_success_notification` you merge `dag_json` and `config_params` so it would be clearer to leave the helper function alone and instead change the call to it